deploy/base/deployment.yaml

```diff
@@ -190,17 +189,19 @@ spec:
           volumeMounts:
             - name: model-config
               mountPath: /datacommons/model
+          startupProbe:
+            httpGet:
+              path: /healthz
```

Comment:
Just double checking, I'm assuming /healthz exists in the nl server.

Reply:
Yes, it exists already